src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -1231,4 +1233,80 @@ private static List<Map<String, Object>> sortByValue(final List<Map<String, Obje
+
+        setUpTestData();
+        String bearerToken = inviteUser(puiFinanceManager, intActiveOrgId, generateRandomEmail(),
+                "firstName2", "lastName2");
+
+        PbaRequest deletePbaRequest = new PbaRequest();
+        deletePbaRequest.setPaymentAccounts(Set.of("PBA0000021", "PBA0000022", "PBA0000023"));
+
+        professionalApiClient.deletePaymentAccountsOfOrganisation(deletePbaRequest,
+                professionalApiClient.getMultipleAuthHeaders(bearerToken), BAD_REQUEST);
+
+        log.info("deletePbaOfExistingOrganisationShouldBeForbiddenWhenLDOff :: END");
+    }
+
+    public String inviteUser(String role,
+                             String orgId,
+                             String email,
+                             String firstName,
+                             String lastName) {
+        List<String> userRoles = new ArrayList<>();
+        userRoles.add(role);
+        NewUserCreationRequest newUserCreationRequest = createUserRequest(userRoles);
+        newUserCreationRequest.setEmail(email);
+        String bearerToken = idamOpenIdClient.getExternalOpenIdToken(puiUserManager,
+                firstName, lastName, email);
+
+        Map<String, Object> pumInternalUserResponse = professionalApiClient
+                .addNewUserToAnOrganisation(orgId, hmctsAdmin, newUserCreationRequest, CREATED);
+        assertThat(pumInternalUserResponse.get("userIdentifier")).isNotNull();
+        return bearerToken;
+    }
+
+    @Test
+    @ToggleEnable(mapKey = "OrganisationExternalController.deletePaymentAccountsOfOrganisation", withFeature = true)
+    @ExtendWith(FeatureToggleConditionExtension.class)
+    void deletePbaOfExistingOrganisationShouldBeSuccess() {
+        log.info("deletePbaOfExistingOrganisationShouldBeSuccess :: STARTED");
+
+        organisationCreationRequest = createOrganisationRequest()
+                .superUser(aUserCreationRequest()
+                        .firstName("firstName")
+                        .lastName("lastName")
+                        .email(generateRandomEmail())
+                        .build())
+                .paymentAccount(Set.of("PBA".concat(RandomStringUtils.randomAlphanumeric(7)),
+                        "PBA".concat(RandomStringUtils.randomAlphanumeric(7)),
+                        "PBA".concat(RandomStringUtils.randomAlphanumeric(7))))
+                .status("ACTIVE")
+                .build();
+        String intActiveOrgId = createAndActivateOrganisationWithGivenRequest(organisationCreationRequest,
+                hmctsAdmin);
+
+        String bearerToken = inviteUser(puiFinanceManager, intActiveOrgId, generateRandomEmail(),
+                "firstName2", "lastName2");
+
+        PbaRequest deletePbaRequest = new PbaRequest();
+        deletePbaRequest.setPaymentAccounts(organisationCreationRequest.getPaymentAccount());
+
+        professionalApiClient.deletePaymentAccountsOfOrganisation(deletePbaRequest,
```

Comment:
this line making a call to existing external endpoint, so this test is invalid. make change to call new internal endpoint with prd-admin user